CHANGELOG.md

```diff
@@ -19,14 +19,35 @@
 
 # Table of Contents
 
+- [2.1.1](#211)
 - [2.1.0](#210)
 - [2.0.0](#200)
 - [1.5.0](#150)
 - [1.0.0](#100)
 
+
+# 2.1.1
+
+This release mainly adds test cases and bugfix for `manager api`.
```

Comment:
`Manager API 2.1.1` should be used with `APISIX 2.1`. 

It is not recommended to use with other APISIX versions.